src/Common/tests/System/Net/Http/Http2LoopbackServer.cs

```diff
@@ -401,7 +402,7 @@ private static int EncodeString(string value, Span<byte> headerBlock)
             new HttpHeaderData("if-modified-since", ""),
             new HttpHeaderData("if-none-match", ""),
             new HttpHeaderData("if-range", ""),
-            new HttpHeaderData("if-unmodifiedsince", ""),
+            new HttpHeaderData("if-unmodified-since", ""),
```

Comment:
Good catch.

Reply:
I would not see it. Test start failing after enabled on HTTP/2